src/pyOpenMS/pxds/Precursor.pxd

```diff
@@ -23,8 +23,15 @@ cdef extern from "<OpenMS/METADATA/Precursor.h>" namespace "OpenMS":
         Precursor(Precursor &) except + nogil 
 
         libcpp_set[ActivationMethod] getActivationMethods() except + nogil  # wrap-doc:Returns the activation methods
+        libcpp_vector[String] getActivationMethodsAsString() except + nogil  # wrap-doc:Returns the activation methods as strings
```

Comment:
Still confusing IMHO. What is the difference between this and "All"?

Reply:
each precursor can have multiple activation methods and that method already existed on the c++ side.

Reply:
that was before "getAllActivationMethods" was added. I think now it needs additional clarification.